docs/interfaces.md

```diff
@@ -463,6 +463,13 @@ trusted apps.
 
 * Auto-Connect: no
 
+### timedate-control
+
+Can modify time/date exposed via D-Bus by systemd-timedated and read all
+all properties of /org/freedesktop/timedate1 D-Bus object.
+
+* Auto-Connect: yes
+
 ### timeserver-control
 
 Can manage timeservers directly separate from ``config core``.
```

Comment:
@jdstrand apologies for forgetting about this change, I'll address it in a separete PR.